cypress/e2e/common-cartridge/exportCourse.feature

```diff
@@ -0,0 +1,91 @@
+@pr
+@stable_test
+Feature: Course Board - To export a course as common cartridge
+
+    As a teacher I want to export a course as a common cartridge file
+
+    Scenario: Teacher exports course
+        Given I am logged in as a '<teacher>' at '<namespace>' in school '5f2987e020834114b8efd6f8' and have course '681e018089002b0ee99aa0af'
+
+        # export the course
+        When I go to courses overview
+        When I go to course 'CC_Test_Kurs'
+        Then I see course page 'CC_Test_Kurs'
+        When I click on export course button
+        When I click on dialog next button
+        When I click on dialog export button
+        Then I have a file exported with pattern 'CC_Test_Kurs-.+\.imscc' and rename it
+
+        # Further validation on file
+        Given the exported file is an archive and extracted
```

Comment:
@pr should not be added now, but after seeing the stablity, we assign @pr and @group tags later.